libcaf_core/caf/telemetry/metric_registry.hpp

```diff
@@ -471,6 +471,12 @@ class CAF_CORE_EXPORT metric_registry {
       visit_family(f, ptr.get());
   }
 
+  // -- modifiers --------------------------------------------------------------
+
+  /// Takes ownership of all metric families in `other`.
+  /// @pre `other` *must not* contain any duplicated metric family
```

Comment:
What does "duplicated" mean here; as in `other` cannot contain the same metric twice or any metric that is present in `this` should not be in `other` (i.e., distinct sets of metric families)?

Reply:
The latter. A registry cannot contain the same metric twice.

Reply:
I can't make up my mind if this could be formulated more precisely or if that is even ambiguous in this context. This is the only nitpick I have. If you are ok with this, feel free to merge it.